### DISCUSSION TITLE
Direct Buy Now / Multi Cart functionality

### DISCUSSION THREAD

**Discussion by pavitra93** [co-cart/co-cart#400] [General]
Hello Dev, 

Firstly thanks for a beautiful headless plugin making life easier. I wanted how can I achieve direct buy now of a single product in the cart. This means multiple cart for 1 user. Any plan for such feature in near future. 

Thanks.

**Comment by pavitra93** [comment]
Also, what if I want to use my custom "custom/v2" namespace for all your API instead of "cocart/v2". Is there any filter available for such?

Thanks

**Reply by seb86** [reply]
No there is no filter for that.

**Comment by seb86** [comment]
> Hello Dev,
> 
> Firstly thanks for a beautiful headless plugin making life easier. I wanted how can I achieve direct buy now of a single product in the cart. This means multiple cart for 1 user. Any plan for such feature in near future.
> 
> Thanks.

I appreciate you saying so. Multi cart functionality is already a feature but not for 1 user. If you mean handling carts for your customers for them then that is being worked on.

**Reply by pavitra93** [reply]
1. I didn't understand what difference cocart makes between users & customers? 
2. Can a customer have multiple carts?

**Reply by seb86** [reply]
1. In terms of a cart session the users and customers are the same.
2. Customers can only have 1 cart.
